### PR TITLE
Geimaj/nft transfers cleanup

### PR DESCRIPTION
**Pull request by Geimaj**
This fixes the error in updating the ERC20 transaction status as we discussed yesterday

### CODE REVIEW

File: src/redux/actions/transactions.ts [hunk 1]
```diff
@@ -140,7 +146,11 @@ export function handlePolkadotTransactionEvents(
   }
 
   if (result.status.isInBlock) {
-    let nonce = result.events[0].event.data[0].toString();
+    let nonce = '0';
```
Comment (musnit) [inline src/redux/actions/transactions.ts:149]: i'm not following 100% why this is done? can you explain why this change?

Comment (Geimaj) [reply src/redux/actions/transactions.ts]: This was to correctly read the nonce from eth and nft transactions but is unrelated to the ERC20 error so I have removed it
